### PR TITLE
[fix](docs) Fix broken graph in en/broker-load-manual.md 

### PR DESCRIPTION
**Pull request by bowenliang123**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

## Further comments

Fix the broken graph in `en/docs/data-operate/import/import-way/broker-load-manual.md`, by reusing the intact version in `zh`.

Before: (see https://doris.apache.org/docs/dev/data-operate/import/import-way/broker-load-manual)
image

After:

image

### CODE REVIEW

**Comment by bowenliang123** [comment]
This PR is fixing a bug in docs introduced at https://github.com/apache/doris/pull/8757.
cc @hf200012

File: docs/en/docs/data-operate/import/import-way/broker-load-manual.md [hunk 1]
```diff
@@ -43,38 +43,38 @@ After the user submits the import task, FE will generate the corresponding Plan
...
-````
+```
                  +
                  | 1. user create broker load
                  v
             +----+----+
-            | |
-            | FE |
-            | |
+            |         |
```
Comment (carlvinhust2012) [inline docs/en/docs/data-operate/import/import-way/broker-load-manual.md:51]: the Chinese version may has the same problem, should fix them together.

Comment (bowenliang123) [reply docs/en/docs/data-operate/import/import-way/broker-load-manual.md:51]: Have check the following links and the Chinese version is good. 
- https://github.com/apache/doris/blob/master/docs/zh-CN/docs/data-operate/import/import-way/broker-load-manual.md#%E5%9F%BA%E6%9C%AC%E5%8E%9F%E7%90%86
- https://doris.apache.org/zh-CN/docs/dev/data-operate/import/import-way/broker-load-manual/

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**